src/components/AddressSearch.js

```diff
@@ -128,6 +134,10 @@ const AddressSearch = (props) => {
             country: '',
         };
 
+        if (country !== CONST.COUNTRY.US) {
```

Comment:
And please a comment here explaining why we override `values.state` if country is not USA